environments/prod/shutter.tfvars

```diff
@@ -0,0 +1,15 @@
+# Common variables - do not change these
+
+env                 = "sbox"
```

Comment:
env sbox?

I thought this wasn't needed anymore, all apps get a storage account and you just need to add the content in the pipeline

Reply:
Roger that, i'll close it now.